app/scripts/directives/fromFile.js

```diff
@@ -157,6 +157,10 @@ angular.module("openshiftConsole")
               openTemplateProcessModal();
             // Else if any resources already exist
             } else if (!_.isEmpty($scope.updateResources)) {
+              // If the updated resource is a Template, set the process option to false
+              if ($scope.updateResources.length === 1 && $scope.updateResources[0].kind === "Template") {
+                $scope.templateOptions.process = false;
```

Comment:
When you are updating an existing Template we dont show the dialog in which user can pick if he wants to process or save(or both) the Template, only the one in which user has to confirm that the new one will replace the old one. After he hits `Replace` he should be redirected to the overview and not to the fromTemplate page as it is now.